include/ocpp/v201/smart_charging.hpp

```diff
@@ -38,7 +44,8 @@ class SmartChargingHandler {
 
     std::shared_ptr<ocpp::v201::DatabaseHandler> database_handler;
     // cppcheck-suppress unusedStructMember
-    std::vector<ChargingProfile> charging_profiles;
+    std::vector<EvseProfile> charging_profiles;
```

Comment:
Would an `std::map` with the evse_id and a vector of charging profiles make sense here? That way we don't need the specific type for `EvseProfile` and we can search a bit quicker too.

Reply:
Ah, yes, you're correct.